packages/slate-editor/src/components/Button/Button.module.scss

```diff
@@ -85,7 +85,29 @@
             background: rgba(255, 255, 255, 0.24);
         }
 
-        &--disabled,
+        .--disabled,
+        &:disabled {
+            opacity: 0.4;
+        }
+    }
+
+    &--secondary {
+        background: $grey-200;
+        color: #4b5563;
+        padding: $spacing-1 - $border-size;
+
+        &:not(&--disabled, &:disabled) {
```

Comment:
`&--disabled`

I think this one is not aligned with other `--disabled` classes.

Reply:
@aspirisen what about this?